report/src/main/webapp/assets/sass/_toolbar.scss

```diff
@@ -235,6 +235,6 @@
   display: inline-block;
 }
 
-.preformatted {
-  font-family: monospace;
```

Comment:
Is this a desired change?
I think that it can cause some regression resulting in few visual glitches. For example here:
![2018-02-26_16h04_26](https://user-images.githubusercontent.com/35067043/36677771-57656dbc-1b0f-11e8-97a0-c1c589fb5815.png)
![2018-02-26_16h04_44](https://user-images.githubusercontent.com/35067043/36677781-5ab3ecb4-1b0f-11e8-8dbf-72fe5d23915f.png)

Reply:
This class is back in css now.